robots/preservation/update_moab.rb

```diff
@@ -0,0 +1,25 @@
+
+module Preservation
+  # Robot either creates or updates moab version then places it into the Moab directory.
+  class UpdateMoab < Base
+    ROBOT_NAME = 'update-moab'.freeze
+
+    def initialize(opts={})
+      super(REPOSITORY, WORKFLOW_NAME, ROBOT_NAME, opts)
+    end
+
+    def perform(druid)
```

Comment:
I kept the debug log statements, thinking they might be useful as we figure out how to get the robots running and hooked up to resque and robot-master and all that.   I can let this idea go ...